Expensify/App
# fix: prevent chat getting scrolled on closing image modal

### Details


This PR fixes an issue where the chat was auto scrolled when an image attachment modal is closed on web browsers. 

### Fixed Issues

$ https://github.com/Expensify/App/issues/6806

### Tests


- Open a chat with image attachments
- scroll the chat so that image thumbnail is only partially visible at the top or bottom of the chat screen
- tap on the visible area of image to open the image attachment modal
- close the modal
- verify that the chat retains its scroll position and there is no sudden movement

### QA Steps


- Open a chat with image attachments
- scroll the chat so that image thumbnail is only partially visible at the top or bottom of the chat screen
- tap on the visible area of image to open the image attachment modal
- close the modal
- verify that the chat retains its scroll position and there is no sudden movement

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/6880914/152600146-ac5a7737-a268-4bd4-aa2d-1016d912482b.mp4

#### Mobile Web


https://user-images.githubusercontent.com/6880914/152600410-64c90fc9-9760-42c3-b9c4-535b814ebaf6.mp4



#### Desktop


https://user-images.githubusercontent.com/6880914/152600341-53c9eceb-b8d6-4016-ab08-177a183531bd.mp4


#### iOS


Not applicable

#### Android


Not applicable
